middlewares/multer.js

```diff
@@ -0,0 +1,57 @@
+import multer from "multer";
+import fs from "fs";
+import path from "path";
+import Jimp from "jimp";
+import {
+  multerStorage,
+  extensionWhiteList,
+  mimetypeWhiteList,
+} from "#config/multer.js";
+
+export const uploadMiddleware = multer({
+  storage: multerStorage,
+  fileFilter: async (req, file, cb) => {
+    const extension = path.extname(file.originalname).toLowerCase();
+    const mimetype = file.mimetype;
+    if (
+      !extensionWhiteList.includes(extension) ||
+      !mimetypeWhiteList.includes(mimetype)
+    ) {
+      return cb(null, false);
+    }
+    return cb(null, true);
+  },
+  limits: { fileSize: 1024 * 1024 * 5 }, // 5MB
+});
+
+export const isImageAndTransform = async (path, filename) =>
```

Comment:
Ta funkcja to to trochę Mix callback spromises i async/await. Warto to opisać w jednym stylu  :)

Reply:
Poprawione :)